tools/Azure.Mcp.Tools.Aks/src/AksSetup.cs

```diff
@@ -28,9 +29,12 @@ public void RegisterCommands(CommandGroup rootGroup, ILoggerFactory loggerFactor
         // Create AKS subgroups
         var cluster = new CommandGroup("cluster", "AKS cluster operations - Commands for listing and managing AKS clusters in your Azure subscription.");
         aks.AddSubGroup(cluster);
+        var nodepool = new CommandGroup("nodepool", "AKS node pool operations - Commands for listing and managing AKS node pools for an AKS cluster.");
```

Comment:
I'll approve the PR, but we're now reviewing all existing tools to see if we can consolidate all the list and get tools together with a new switch.

Reply:
Cluster and node pool could probably also be a switch to one common tool, to help keep the tool count down?